frontend/src/pages/SearchUISearchPage.js

```diff
@@ -210,6 +214,18 @@ class SearchUISearchPage extends React.Component<SearchUISearchPageProps, Search
         </div>
       </div>
     );
+    const pageContents = showContents ? (
```

Comment:
Is there a way for this to look less disruptive? When I click Go I will see the page go blank except for the spinner and then show the results...
What do you think of just drawing the spinny thing on top of a translucent gray overlay while keeping the previous results semi-visible until the new ones are fetched?